src/js/tabs/osd.js

```diff
@@ -1107,6 +1115,12 @@ OSD.chooseFields = function () {
         F.RTC_DATE_TIME
       ]);
     }
+      if (semver.gte(CONFIG.apiVersion, "1.41.0")) {
```

Comment:
This shouldn't be here, this should be in the `else` part below.

Reply:
I added it to the else, but removed it from the other field. Having them in both fields lists them two times.